tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -167,6 +167,7 @@ def generate_files(list, args):
                           '" target="build\\native\\include" />')
 
     if includes_winml:
+        mlop_path = 'onnxruntime\\core\\providers\\dml\\dmlexecutionprovider\\inc\\mloperatorauthor.h'
```

Comment:
using the portable `os.path.join('onnxruntime','core','providers','dml','dmlexecutionprovider','inc','mloperatorauthor.h')` prevents backslash escaping

Reply:
But it doesn't need be portable, winml is just for Windows.

Reply:
Agreed, that is why I didn't request a change but instead just hinted the Pythonic way to achieve the same goal :) Escaping characters are often sources of bugs.